server/rest/src/test/java/org/infinispan/rest/resources/LocalClusterResourceTest.java

```diff
@@ -0,0 +1,89 @@
+package org.infinispan.rest.resources;
+
+import org.assertj.core.api.Assertions;
+import org.infinispan.client.rest.RestClient;
+import org.infinispan.client.rest.RestResponse;
+import org.infinispan.client.rest.configuration.RestClientConfigurationBuilder;
+import org.infinispan.commons.dataconversion.internal.Json;
+import org.infinispan.commons.test.TestResourceTracker;
+import org.infinispan.factories.impl.BasicComponentRegistry;
+import org.infinispan.manager.EmbeddedCacheManager;
+import org.infinispan.rest.TestClass;
+import org.infinispan.rest.assertion.ResponseAssertion;
+import org.infinispan.rest.authentication.impl.BasicAuthenticator;
+import org.infinispan.rest.helper.RestServerHelper;
+import org.infinispan.rest.resources.security.SimpleSecurityDomain;
+import org.infinispan.security.actions.SecurityActions;
+import org.infinispan.server.core.DummyServerStateManager;
+import org.infinispan.server.core.ServerStateManager;
+import org.infinispan.test.SingleCacheManagerTest;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+import static org.infinispan.commons.util.concurrent.CompletionStages.join;
+import static org.infinispan.rest.resources.AbstractRestResourceTest.ADMIN;
+import static org.infinispan.rest.resources.AbstractRestResourceTest.REALM;
+import static org.infinispan.rest.resources.AbstractRestResourceTest.USER;
+import static org.infinispan.rest.resources.ClusterResource.CACHE_MANAGER_STATUS;
+import static org.infinispan.rest.resources.ClusterResource.MEMBERS;
+import static org.infinispan.rest.resources.ClusterResource.NODE_ADDRESS;
+import static org.infinispan.rest.resources.ClusterResource.PHYSICAL_ADDRESSES;
+import static org.infinispan.rest.resources.ClusterResource.ROLLING_UPGRADE;
+import static org.infinispan.rest.resources.ClusterResource.VERSION;
+
+@Test(groups = "functional", testName = "rest.ClusterResourceTest")
```

Comment:
This test class needs to killl the RestServer it starts in the @AfterClass

Reply:
done